Index: .htaccess
===================================================================
RCS file: /cvs/drupal/drupal/.htaccess,v
retrieving revision 1.90.2.4
diff -u -p -r1.90.2.4 .htaccess
--- .htaccess	7 Dec 2009 12:00:40 -0000	1.90.2.4
+++ .htaccess	8 Jan 2010 00:52:14 -0000
@@ -93,8 +93,10 @@ DirectoryIndex index.php
   # To redirect all users to access the site WITHOUT the 'www.' prefix,
   # (http://www.example.com/... will be redirected to http://example.com/...)
   # uncomment and adapt the following:
-  # RewriteCond %{HTTP_HOST} ^www\.example\.com$ [NC]
-  # RewriteRule ^(.*)$ http://example.com/$1 [L,R=301]
+   RewriteCond %{HTTP_HOST} ^www\.rocktreesky\.com$ [NC]
+   RewriteRule ^(.*)$ http://rocktreesky.com/$1 [L,R=301]
+   RewriteCond %{HTTP_HOST} ^www\.25mileradius\.com$ [NC]
+   RewriteRule ^(.*)$ http://25mileradius.com/$1 [L,R=301]
 
   # Modify the RewriteBase if you are using Drupal in a subdirectory or in a
   # VirtualDocumentRoot and the rewrite rules are not working properly.
@@ -111,6 +113,15 @@ DirectoryIndex index.php
   RewriteCond %{REQUEST_FILENAME} !-d
   RewriteCond %{REQUEST_URI} !=/favicon.ico
   RewriteRule ^(.*)$ index.php?q=$1 [L,QSA]
+  
+  # CUSTOM ADDITION: MINT FEEDBACK MODS
+  RewriteCond %{QUERY_STRING} !FB_secret=a4ae32$
+  RewriteRule ^rss.xml/?$ /mint/pepper/hansvankilsdonk/feedback/tracker.php?FB_feed=http://rocktreesky.com/rss.xml&FB_secret=a4ae32&%{QUERY_STRING} [L]
+  RewriteCond %{QUERY_STRING} !FB_secret=a4ae32$
+  RewriteRule ^tags/geek/drupal/feed/?$ /mint/pepper/hansvankilsdonk/feedback/tracker.php?FB_feed=http://rocktreesky.com/tags/geek/drupal/feed&FB_secret=a4ae32&%{QUERY_STRING} [L]
+
+  # CUSTOM ADDITION: Webmail redirect
+  Redirect /webmail https://webmail.tuffmail.net/horde-3.1.1/imp/
 </IfModule>
 
 # $Id: .htaccess,v 1.90.2.4 2009/12/07 12:00:40 goba Exp $
